db-migrations/000005_adding_owners.up.sql

```diff
@@ -0,0 +1,22 @@
+-- schema changes for account owners and champions
```

Comment:
@ensary this is a many-many design similar to what we discussed, one aspect of this though is that owner is technically `one to many`, since an account can have only one owner. This is not currently modeled with these many-many tables and I'll need to enforce. 

I am considering adding a FK owner column to the aws_accounts table but one problem with that will be populating that column and is going to be more difficult in some ways.

Reply:
`Any of owner/champions can be null (but should not)`
Makes me not want to use a foreign key though